airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -203,6 +203,27 @@ class FutureStateConfig(BaseConfig):
     bypass_reason: Optional[str]
 
 
+class CheckpointingStrategies(str, Enum):
```

Comment:
can you clarify how a developer should decide on which strategy to use?

My understanding is we should use `use_state_variation` in all cases except if there isn't enough state messages.

Alternative: Can we avoid the configuration option and fallback to `use_latest_state` if fewer than 3 state messages were read?

Reply:
Yes, we can. Added logic to fallback automatically

Reply:
removed